mcserver/objects/packet_handler.py

```diff
@@ -118,10 +119,12 @@ def decode_plugin_message(cls, msg: ClientMessage):
                     msg.protocol_version,
                     "???"),
                 "protocol": msg.protocol_version
-            }
+            },
         }
-        # TODO: Server icon
+        if ServerCore.options["favicon"]:
```

Comment:
This should use the `server.icon` file instead, similar to the official server implementation

Reply:
got it.